### PR TITLE
chore(notes): update release notes for 2019.04

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -79,7 +135,7 @@
 ### Layout
 *  Updated docs [(#1885](https://github.com/patternfly/patternfly-next/pull/1885))
 
-# PatternFly RC2 (2019-05-28)
+## 2019.01 - RC2 - release notes (2019-05-28)
```
Comment (redallen) [inline RELEASE-NOTES.md]: Can you remove the first dash to be consistent with "2019.02 RC2.1 release notes"?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,4 +1,60 @@
-# PatternFly Albatross Milestone (2019-07-02)
+## 2019.04 release notes (2019-07-24)
```
Comment (redallen) [inline RELEASE-NOTES.md]: Why are these `h2`s now instead of `h1`s like they used to be?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+- @patternfly/patternfly@2.23.0
+
```
Comment (redallen) [inline RELEASE-NOTES.md]: Add `## Components` subtitle.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,4 +1,60 @@
...
+**About modal:** Updated break-word on content area instead of break-all ([#2035](https://github.com/patternfly/patternfly-next/pull/2035))
```
Comment (redallen) [inline RELEASE-NOTES.md]: @rachael-phillips wants these to have `AutoLinkHeader` anchors on -org, so please make them `h3`s like `### About modal` like I did for the other notes.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,4 +1,60 @@
...
+
+**Divider:** Added divider component ([#2080](https://github.com/patternfly/patternfly-next/pull/2080))
+
+**Drawer:** Added drawer component ([#2069](https://github.com/patternfly/patternfly-next/pull/2069))
+
+**Dropdown:**
+- Added space-between when width of dropdown grows ([#2050](https://github.com/patternfly/patternfly-next/pull/2050))
+- Centered plain button content ([#2071](https://github.com/patternfly/patternfly-next/pull/2071))
+    
+**Experimental features:** Added experimental feature support ([#2031](https://github.com/patternfly/patternfly-next/pull/2031))
```
Comment (mcoker) [inline RELEASE-NOTES.md]: should this be under an "infrastructure" heading like the last release?

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,4 +1,60 @@
...
+* Added support for right and top alignment ([#2081](https://github.com/patternfly/patternfly-next/pull/2081))
+
+**Charts**
+- Adjusted chart vars for react-charts ([#2020](https://github.com/patternfly/patternfly-next/pull/2020), [#2024](https://github.com/patternfly/patternfly-next/pull/2024))
+- Adjusted 3 chart tooltip color vars and adds two new ones. This will ensure tooltips can be seen over the current background color. ([#2038](https://github.com/patternfly/patternfly-next/pull/2038))    
+- Added individual padding vars for donut charts ([#2068](https://github.com/patternfly/patternfly-next/pull/2068))
+
+**Chip:** updated padding on chip label ([#2063](https://github.com/patternfly/patternfly-next/pull/2063))
+
+**Chore:** Updated release notes for latest milestones ([#2014](https://github.com/patternfly/patternfly-next/pull/2014))
```
Comment (mcoker) [inline RELEASE-NOTES.md]: Maybe under a "docs" or "other" heading like the previous releases use?

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,4 +1,60 @@
...
+**App launcher**
+* Added button example to menu item ([#2006](https://github.com/patternfly/patternfly-next/pull/2006)) 
+* Added support for right and top alignment ([#2081](https://github.com/patternfly/patternfly-next/pull/2081))
+
+**Charts**
+- Adjusted chart vars for react-charts ([#2020](https://github.com/patternfly/patternfly-next/pull/2020), [#2024](https://github.com/patternfly/patternfly-next/pull/2024))
+- Adjusted 3 chart tooltip color vars and adds two new ones. This will ensure tooltips can be seen over the current background color. ([#2038](https://github.com/patternfly/patternfly-next/pull/2038))    
+- Added individual padding vars for donut charts ([#2068](https://github.com/patternfly/patternfly-next/pull/2068))
+
+**Chip:** updated padding on chip label ([#2063](https://github.com/patternfly/patternfly-next/pull/2063))
```
Comment (mcoker) [inline RELEASE-NOTES.md]: in the previous notes, a single item under a component is it's own list with 1 item. Just want to make sure doing it this way is on purpose?

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,4 +1,60 @@
...
+
+**Chore:** Updated release notes for latest milestones ([#2014](https://github.com/patternfly/patternfly-next/pull/2014))
+
+**Divider:** Added divider component ([#2080](https://github.com/patternfly/patternfly-next/pull/2080))
+
+**Drawer:** Added drawer component ([#2069](https://github.com/patternfly/patternfly-next/pull/2069))
+
+**Dropdown:**
```
Comment (mcoker) [inline RELEASE-NOTES.md]:
```suggestion
**Dropdown**
```

**Comment by redallen** [comment]
Closing in favor of #2111 .

**Pull request closed without merging**